### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by jennyfolkesson**
Added Travis CI and fixed tests.

### CODE REVIEW

File: tests/preprocessing/generate_masks_tests.py [hunk 1]
```diff
@@ -207,13 +207,13 @@ def _get_name(ch_idx, sl_idx, time_idx, pos_idx):
...
-            index_col=0
+            os.path.join(mask_gen_inst.get_mask_dir(), 'frames_meta.csv'),
+            index_col=0,
         )
         # pos1: 8 slices, pos2: 3 slices
-        exp_len = 22
+        exp_len = 8 + 3
         nose.tools.assert_equal(len(frames_meta), exp_len)
-        mask_fnames = frames_meta['file_name'].tolist()[11:22]
+        mask_fnames = frames_meta['file_name'].tolist()
```
Comment (anithapk) [inline tests/preprocessing/generate_masks_tests.py]: 👍

File: tests/preprocessing/tile_uniform_images_tests.py [hunk 2]
```diff
@@ -87,6 +93,17 @@ def setUp(self):
...
+                      'im_c001_z016_t005_p007.png',
+                      'im_c001_z017_t005_p007.png']
+        self.exp_fnames = [os.path.join(self.temp_path, fname)
+                      for fname in exp_fnames]
+        self.exp_tile_indices = [
+            [0, 5, 0, 5], [0, 5, 4, 9], [0, 5, 6, 11],
+            [10, 15, 0, 5], [10, 15, 4, 9], [10, 15, 6, 11],
+            [4, 9, 0, 5], [4, 9, 4, 9], [4, 9, 6, 11],
+            [8, 13, 0, 5], [8, 13, 4, 9], [8, 13, 6, 11],
+        ]
```
Comment (anithapk) [inline tests/preprocessing/tile_uniform_images_tests.py]: 👍

File: tests/preprocessing/tile_uniform_images_tests.py [hunk 3]
```diff
@@ -325,25 +315,18 @@ def test_get_tile_args(self):
...
         )
-        exp_input_fnames = ['im_c001_z015_t005_p007.npy',
-                            'im_c001_z016_t005_p007.npy',
-                            'im_c001_z017_t005_p007.npy']
-        exp_fnames = [os.path.join(mask_dir, fname)
-                      for fname in exp_input_fnames]
-
-        nose.tools.assert_list_equal(list(cur_args[0]), exp_fnames)
+        nose.tools.assert_list_equal(list(cur_args[0]), self.exp_fnames)
         # flat field fname is None
```
Comment (anithapk) [inline tests/preprocessing/tile_uniform_images_tests.py]: Sorry, I should've added a note here! get_crop_tile_args with task_type=tile is used for two cases: 1) Mask image with a min_fraction to identify the indices to keep. 2) Grayscale image for tiling and keep all tiles. 
The two cases here were for the above two usages. I should've made this clearer. The actual returned indices are tested in tests for tile_utils. Thanks for fixing this.

Comment (jennyfolkesson) [reply tests/preprocessing/tile_uniform_images_tests.py]: I was just aiming to get them to pass without looking too much into their purpose, I hope I didn't change the meaning of your tests!

**Pull request merged**